lib/core/output_sink/es_sink.rb

```diff
@@ -56,6 +56,10 @@ def delete_multiple(ids)
       ids.each { |id| delete(id) }
     end
 
+    def ingestion_stats
+      {}
```

Comment:
To be implemented by @artem-shelkovnikov  in https://github.com/elastic/connectors-ruby/pull/389